account_due_list_payment_mode/__openerp__.py

```diff
@@ -1,11 +1,12 @@
-# -*- encoding: utf-8 -*-
+# -*- coding: utf-8 -*-
 
 {
     "name": "Payment due list with payment mode",
     "version": "9.0.1.0.0",
     "category": "Generic Modules/Payment",
     "author": "Obertix Free Solutions, "
               "Tecnativa, "
+              "Haevas, "
```

Comment:
As your contribution is very small, you shouldn't add yourself to the author's list